frontend/src/pages/PropertyDetails.jsx

```
@@ -0,0 +1,403 @@
+                                        </>
+                                    )}
+                                    
+                                    <button
+                                        onClick={handleBooking}
+                                        disabled={!bookingDates.checkIn || !bookingDates.checkOut || property.status !== 'available'}
+                                        className={`w-full py-3 rounded-md text-white font-semibold ${
+                                            property.status === 'available' && bookingDates.checkIn && bookingDates.checkOut
+                                                ? 'bg-blue-600 hover:bg-blue-700 transition duration-300'
+                                                : 'bg-gray-400 cursor-not-allowed'
+                                        }`}
+                                    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)